internal/cmd/log-file-metric-exporter/main.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"flag"
+	"github.com/ViaQ/logerr/log"
+	"os"
+	"path"
+	"strings"
+	"net/http"
+	"github.com/openshift/cluster-logging-operator/internal/pkg/symnotify"
+        "github.com/prometheus/client_golang/prometheus"
+        "github.com/prometheus/client_golang/prometheus/promhttp"
+)
+
+
+
+
+var (
+ verbosity int = 0
+ namespace string = "unknown"
```

Comment:
I don't believe these are needed

Reply:
@jcantrill we need to initialise these variables somewhere right ?

Reply:
i dont think we need them at all because they are values that will always be known from the file name pattern